.gitignore

```
@@ -89,4 +89,4 @@ venv/
 .spyderproject
 
 # Rope project settings
-.ropeproject
+.ropeproject
```

We need a newline at the end of the file.
```suggestion
.ropeproject

```

---

Alright, let me add it.